docs/v3/core/v3.0.rst

```diff
@@ -647,18 +647,6 @@ that chunk, where "%" is the modulo operator. For example, if a
 is contained within the chunk at grid index (1, 7, 2) and has coordinates
 (2, 10, 100) within that chunk.
 
-The identifier for chunk with grid index (``k``, ``j``, ``i``, ...) is
-formed by taking the initial prefix ``c``, and appending for each dimension:
-
-- the ``separator`` character specified within the ``chunk_grid`` metadata object (see
-  the section on `Array metadata`_ below), followed by,
-
-- the ASCII decimal string representation of the chunk index within that dimension.
-
-For example, in a 3 dimensional array, with a separator of ``/`` the identifier
-for the chunk at grid index (1, 23, 45) is the string "c/1/23/45".  With a
-separator of ``.``, the identifier is the string "c.1.23.45".
-
```

Comment:
Please add a reference to the chunk_key_encoding metadata section here.

Reply:
Done.